### PR TITLE
Update revoke-ome-encrypted-mail.md

### LINKED ISSUE #12217: No Message encryption report in the Microsoft Purview compliance portal
In the Microsoft Purview compliance portal there is no "Message encryption report"

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 10cd0b42-9f8f-265f-1f4b-f5b73f811cfc
* Version Independent ID: f400b434-c016-7ee5-ef38-b57b90dcab95
* Content: [Revoke email encrypted by Advanced Message Encryption - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/revoke-ome-encrypted-mail?view=o365-worldwide#encrypted-emails-that-you-can-revoke)
* Content Source: [microsoft-365/compliance/revoke-ome-encrypted-mail.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/revoke-ome-encrypted-mail.md)
* Service: **o365-seccomp**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

### PR DESCRIPTION
**Pull request by msbemba**
Updated the portal

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12217

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/revoke-ome-encrypted-mail.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ There are multiple ways to find the Message ID of the email that you want to rev
 
 2. Once you've located the email, select it to bring up the **Message trace details** pane. Expand **More Information** to locate the Message ID.
 
-#### To identify the Message ID of the email you want to revoke by using Message Encryption reports in the Microsoft Purview compliance portal
+#### To identify the Message ID of the email you want to revoke by using Message Encryption reports in the Microsoft 365 Defender portal
 
-1. In the Microsoft Purview compliance portal, navigate to the **Message encryption report**. For information on this report, see [View email security reports in the Security &amp; Compliance Center](../security/office-365-security/reports-email-security.md).
+1. In the Microsoft 365 Defender portal, navigate to the **Message encryption report**. For information on this report, see [View email security reports in the Security &amp; Compliance Center](../security/office-365-security/reports-email-security.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/revoke-ome-encrypted-mail.md]:
```suggestion
1. In the Microsoft 365 Defender portal, navigate to the **Message encryption report**. For information about this report, see [View email security reports in the Security &amp; Compliance Center](../security/office-365-security/reports-email-security.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
Closing as per the comment in #12217

**Pull request closed without merging**